pegasus/router.rb

```diff
@@ -110,9 +110,12 @@ def self.set_max_age(type, default)
       settings.template_extnames +
       settings.exclude_extnames +
       ['.fetch']
+    # Note: shared_resources.rb has additional configuration for Sass::Plugin
     Sass::Plugin.options[:cache_location] = pegasus_dir('cache', '.sass-cache')
-    Sass::Plugin.options[:css_location] = pegasus_dir('cache', 'css')
-    Sass::Plugin.options[:template_location] = shared_dir('css')
+    Sass::Plugin.add_template_location(
+      sites_v3_dir('code.org', 'public', 'css'),
```

Comment:
Does putting the `.scss` files in `pegasus/sites.v3/code.org/public/css` alongside regular css files sound good?  Or should we put them in a separate directory?

Reply:
I think for easier access they can live alongside .css files in the `/public/css` directory. Happy to defer to the others though.